docs/source/configuration.rst

```diff
@@ -31,40 +31,90 @@ the ``config`` dictionary or the ``get`` function:
+   'temporary-directory': None}
+   
 
-   >>> dask.config.get("distributed.logging")
-   {
-       'distributed': 'info',
-       'bokeh': 'critical',
-       'tornado': 'critical'
-   }
-
-   >>> dask.config.get('distributed.logging.bokeh')  # use `.` for nested access
-   'critical'
+   >>> import dask.distributed  # populate config with distributed defaults
+   >>> dask.config.config
+   {'array': {'slicing': {'split-large-chunks': None}, 'svg': {'size': 120}},
+   'dataframe': {'shuffle-compression': None},
+   'distributed': {'adaptive': {'interval': '1s',
+                                'maximum': inf,
+                                'minimum': 0,
+                                'target-duration': '5s',
+                                'wait-count': 3},
+                  ...,
+                   'dashboard': {'export-tool': False,
+                                 'graph-max-items': 5000,
+                                 'link': '{scheme}://{host}:{port}/status',
+                                 'prometheus': {'namespace': 'dask'}},
+                   'deploy': {'cluster-repr-interval': '500ms',
+                              'lost-worker-timeout': '15s'},
+                  ...,
+                   'worker': {'blocked-handlers': [],
+                              'connections': {'incoming': 10, 'outgoing': 50},
+                              'daemon': True,
+                              'http': {'routes': ['distributed.http.worker.prometheus',
+                                                  'distributed.http.health',
+                                                  'distributed.http.statics']},
+                              'lifetime': {'duration': None,
+                                           'restart': False,
+                                           'stagger': '0 seconds'},
+                              'memory': {'pause': 0.8,
+                                         'rebalance': {'measure': 'optimistic',
+                                                       'recipient-max': 0.6,
+                                                       'sender-min': 0.3,
+                                                       'sender-recipient-gap': 0.1},
+                                         'recent-to-old-time': '30s',
+                                         'spill': 0.7,
+                                         'target': 0.6,
+                                         'terminate': 0.95},
+                              'multiprocessing-method': 'spawn',
+                              'preload': [],
+                              'preload-argv': [],
+                              'profile': {'cycle': '1000ms',
+                                          'interval': '10ms',
+                                          'low-level': False},
+                              'resources': {},
+                              'use-file-locking': True,
+                              'validate': False}},
+  ...}
```

Comment:
This may be a bad idea, but I'm going to suggest we remove this entire portion for two reasons:

1. Today we actually don't recommend users use `dask.config.config` directly. Instead we recommend using `dask.config.get(...)` for configuration access as it handles `-` and `_` correctly.
2. Displaying hardcoded configuration setting like this will ultimately result in our documentation getting out of date (xref https://github.com/dask/dask/issues/7689)

@ncclementi @jsignell what do you think?

Reply:
Yeah that makes sense to me. I think we should focus on demonstrating what people _should_ do rather than what they _can_ do